openquake/baselib/performance.py

```diff
@@ -103,27 +106,27 @@ def start_time(self):
         """
         return datetime.fromtimestamp(self._start_time)
 
-    # this is used by readinput.get_composite_source_model
-    def write(self, row):
-        """Write a row in the performance file, if any"""
-        if self.perftemp is None:
-            fd, self.perftemp = tempfile.mkstemp(suffix='.csv')
-            os.close(fd)
-        open(self.perftemp, 'a').write('\t'.join(row) + '\n')
-
     def get_data(self):
         """
-        Return a list of strings with the measured operation, time and memory
+        :returns:
+            an array of dtype perf_dt, with the information of the monitor
+            and its children (operation, time_sec, memory_mb, counts)
+
+        NB: at the moment only the direct children are retrieved, i.e.
```

Comment:
You might consider leveraging sphinx directives such as ``note`` or ``warning``, like:
```
.. note::

    at the moment only the direct children are retrieved, i.e.
    get_data is not recursive.
```